Description:
  Use /usr/share/ocropus/scripts/ and /usr/share/ocropus/ as defaults for
  OCROSCRIPTS and OCRODATA.
Author: Jakub Wilk <jwilk@debian.org>

Index: ocropus-0.3.1/ocroscript/ocrotoplevel.cc
===================================================================
--- ocropus-0.3.1.orig/ocroscript/ocrotoplevel.cc	2009-11-26 16:56:18.000000000 +0100
+++ ocropus-0.3.1/ocroscript/ocrotoplevel.cc	2009-11-26 17:16:32.000000000 +0100
@@ -68,10 +68,10 @@
 
 // FIXME the Jamfile isn't passing this flag, so for now, this is a workaround
 #ifndef OCROSCRIPTS
-#define OCROSCRIPTS "/usr/local/share/ocropus/scripts/"
+#define OCROSCRIPTS "/usr/share/ocropus/scripts/"
 #endif
 #ifndef OCRODATA
-#define OCRODATA "/usr/local/share/ocropus/"
+#define OCRODATA "/usr/share/ocropus/"
 #endif
 
 const char *ocroscripts = OCROSCRIPTS;
